lib/services/mail_fetcher.dart

```diff
@@ -109,7 +109,9 @@ class MailFetcher {
                 mimeParts[i].parts![j].contentType!.mediaType.top;
             switch (subPartTopType) {
               case MediaToptype.image:
-                attachments.add(_grabImage(mimeParts[i].parts![j]));
+                if( !mimeParts[i].parts![j].headersList.toString().contains("name=\"ra_0") ) {
```

Comment:
ridealong content still needs to either get created as its own mailpiece, or they need to be combined (with scan text) with their actual mailpiece (so you would see two images and a combined set of text in one view).

we don't want to hide that content altogether

Reply:
fixed it so ride along mailpieces are added, and the mailpiece ids are found so the links can work.  the way the search works they usually get found together